@kiva/kv-components/vue/KvLendCta.vue

```diff
@@ -418,6 +519,10 @@ export default {
 				this.handleCheckout();
 			}
 		},
+		clickPresetButton(selectedDollarAmount) {
+			this.kvTrackFunction('Lending', 'Modify lend amount', selectedDollarAmount, this.loanId, this.loanId);
```

Comment:
Probably we should emit this in order to track an specific event for BP.

Reply:
Emit or pass in tracking category with default of `Lending`

Reply:
A vague memory Greg asked to have `borrower-profile`...

Reply:
Right, we'd pass in `borrower-profile` for the new usage, but the existing CTA component usage still has `Lending`, hence having that be the default of the new category prop (if we go that route instead of emitting).